src/EditorFeatures/.editorconfig

```diff
@@ -0,0 +1,7 @@
+# CSharp IDE Team code style settings
+# Use var as much as possible
+[*.cs]
+csharp_style_var_for_locals = true:suggestion
```

Comment:
Is this easier to define in the core one and then override in the compiler path?